sources/lib/util/steps.mli

```diff
@@ -0,0 +1,12 @@
+type incr_kind = Matching | Omega | Fourier | Uf | Builtin | Ac | Naive of int
```

Comment:
These steps kind **must** be documented, ^^
(as a rule of thumb, any change/additions in interfaces should ensure that the modified parts have proper documentation)